src/mumble/CMakeLists.txt

```diff
@@ -399,32 +399,39 @@ if(bundled-opus)
 	option(OPUS_BUILD_SHARED_LIBRARY "" ON)
 	add_subdirectory("${3RDPARTY_DIR}/opus-src" "${CMAKE_CURRENT_BINARY_DIR}/opus")
 
-	# Shared library on UNIX (e.g. ".so")
-	set_target_properties(opus PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
-	# Shared library on Windows (e.g. ".dll")
-	set_target_properties(opus PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
-
-	# We don't actually link to Opus, because we load it manually at runtime.
-	# We use target_link_libraries() just because it automatically marks "opus"
-	# as dependency and adds the interface include directories to the target.
-	target_link_libraries(mumble PRIVATE opus)
+	if(UNIX)
+		# Shared library on UNIX (e.g. ".so")
+		set_target_properties(opus PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+
+		target_link_libraries(mumble PRIVATE opus)
+	elseif(MSVC)
```

Comment:
This is fine, but I would use `elseif(WIN32)` instead.

We will probably add support for MinGW in future.

Alternatively:
```cmake
	add_dependencies(mumble opus)

	if(WIN32)
		set_target_properties(opus PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})

		target_include_directories(mumble PRIVATE "${3RDPARTY_DIR}/opus-src/include")

	else()
		set_target_properties(opus PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})

		target_link_libraries(mumble PRIVATE opus)
	endif()
```

Reply:
Addressed with latest push